partner_certificate_of_conduct/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017-2018 Onestein (<https://www.onestein.eu>)
+# Copyright 2018 Therp BV (<https://www.therp.nl>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner Certificate of Conduct',
+    'summary': "Adds field 'Certificate of Conduct'",
+    'version': '11.0.0.0.0',
+    'category': 'Web',
```

Comment:
web?  this category is for web interface usability changes. i see other modules using "base" , "extra tools" or "customer  relationship management". I believe the last is most appropriate.